Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Pass/PassDefines.h

```diff
@@ -43,6 +43,9 @@ namespace AZ
         // Rendering -> Idle
         //           -> Queued (Rendering will transition to Queued if a pass was queued with the PassSystem during Rendering)
         //
+        // Any State -> Orphaned  (transition to Orphaned state can be outside the jurisdiction of the pass and so can happen from any state)
+        // Orphaned  -> Queued    (Orphaned state transitions back to idle. 
```

Comment:
nit: no closing parenthesis on comment. Also it's a little confusing the the comment says idle which is a different state than queued. I'm guessing you mean it'll eventually transition through states to get back to idle, but not sure.

Reply:
Thanks for the catch, will definitely fix this (AR failed anyway so have to re-run it)